static/js/recent_topics_ui.js

```diff
@@ -569,7 +593,16 @@ export function complete_rerender() {
         simplebar_container: $("#recent_topics_table .table_fix_head"),
         callback_after_render: revive_current_focus,
         is_scroll_position_for_render,
-        post_scroll__pre_render_callback: set_focus_to_element_in_center,
+        post_scroll__pre_render_callback: () => {
+            message_fetch.maybe_load_older_messages({
```

Comment:
We don't want to do this on every scroll attempt, right?

Reply:
https://github.com/zulip/zulip/pull/19151#issuecomment-876173311 also answers this.